src/main.ts

```
@@ -1,64 +1,44 @@
-    redirectLogsToBot,
-    setupConsoleLogger,
-    setupExitHandler,
     uploadLogsToS3,
+    setupExitHandler,
+    setupConsoleLogger,
 } from './utils/Logger'
+import { server } from './server'
+
+import { JoinError, MeetingParams } from './types'
+
+import { exit } from 'process'
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)